guides/v2.3/graphql/reference/wishlist.md

```diff
@@ -12,14 +12,14 @@ Use Wishlist queries to retrieve information about a customer's wish list.
 
 Attribute | Data type | Description
 --- | --- | ---
-`items` | [Wishlistitem](#wishlistitem) | An array of items in the customer's wish list
+`items` | [WishlistItem](#WishlistItem) | An array of items in the customer's wish list
```

Comment:
I approve the data type rename to initial cap Item, but we should not update the anchor link. Please remove at part of your change.

Reply:
Done